lib/mpl_toolkits/mplot3d/proj3d.py

```diff
@@ -146,6 +144,35 @@ def proj_transform(xs, ys, zs, M):
 transform = proj_transform
 
 
+def _proj_transform_vectors(vecs, M):
+    """Vector version of ``project_transform`` able to handle MaskedArrays.
+
+    Parameters
+    ----------
+    vecs : *x3 np.ndarray or np.ma.MaskedArray
+        Input vectors
+
+    M : 4x4 np.ndarray
+        Projection matrix
```

Comment:
nit:
```suggestion
    vecs : ... x 3 np.ndarray or np.ma.MaskedArray
        Input vectors

    M : 4 x 4 np.ndarray
        Projection matrix
```
Perhaps clearer so the `*` doesn't look like a multiply